### ISSUE TITLE
[ENH]: Is there any way to get the status bar bg color transparent?

### ISSUE THREAD

**Issue by sebalfaro** [2KAbhishek/tmux2k#21]
### Description

Also, just one question, can i set a diff color to active window? Thank for your contributions, the plugin is nice 👍🏽

**Comment by 2KAbhishek** [comment]
No, transparent colors not being supported is unfortunately a limitaion of tmux itself, the plugin won't be able to fix it, what you can do though is pick a colour that is very close to your terminal bg / wallpaper, achieves the same effect, I just use #000000 since I use dark wallpapers

Ya, you can change window list colors with this:
```sh
set -g @tmux2k-window-colors "red black"
```

Please close the issue if this answers your question

**Comment by pdf** [comment]
FYI for anyone coming to look for this, you may use the following to set a transparent bg color (assuming your terminal uses the default background as transparent, most do):

```
set -g @tmux2k-bg-main 'default'
```
